packages/api/src/command/medical/patient/convert-fhir-bundle.ts

```diff
@@ -44,17 +44,20 @@ export async function handleBundleToMedicalRecord({
   dateTo?: string;
   conversionType: ConsolidationConversionType;
 }): Promise<Bundle<Resource>> {
-  const isSandbox = Config.isSandbox();
-
-  if (isSandbox) {
+  if (Config.isSandbox()) {
+    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+    const bucketName = Config.getSandboxSeedBucketName()!;
```

Comment:
Not a huge deal, but we can easily avoid the non-null assertion by returning the bucket name and adding it to the conditional along with `isSandbox`. Makes for a cleaner code and avoids the situation where sandbox is enabled but the bucket was not.

I know it was like this, but since we're touching it might as well make it better 🖖